jenkins/release-workflows/release-notes-generate.jenkinsfile

```
@@ -0,0 +1,171 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+lib = library(identifier: 'jenkins@10.2.2', retriever: modernSCM([
+    $class: 'GitSCMSource',
+    remote: 'https://github.com/opensearch-project/opensearch-build-libraries.git',
+]))
+
+pipeline {
+    options {
+        timeout(time: 2, unit: 'HOURS')
+    }
+
+    agent none
+
+    parameters {
+        string(
+            name: 'INPUT_MANIFEST',
+            description: 'Release version number to fetch input manifest path',
+            trim: true
+        )
+        string(
+            name: 'GIT_LOG_DATE',
+            description: 'in format yyyy-mm-dd, example 2022-07-26.',
+            trim: true
+        )
+        string(
+            name: 'COMPONENTS',
+            description: 'Optional: If this field contains one or more component names (e.g. OpenSearch common-utils ...), will build with "--component COMPONENT_NAME_HERE ...", else build everything in the INPUT_MANIFEST.',
+            trim: true
+        )
+        string(
+            name: 'MODEL_ID',
+            description: 'AWS Bedrock model id to use, defaults to Anthropic Claude 3.7v2',
+            trim: true
+        )
+        string(
+            name: 'MAX_TOKENS',
+            description: 'Max number of tokens to use for supplied model id, deafults to 2000',
+            trim: true
```

Same as above as default?
```suggestion
            name: 'MAX_TOKENS',
            description: 'Max number of tokens to use for supplied model id, defaults to 2000',
            trim: true
```